### DISCUSSION TITLE
Windows 98 SE boots and Sound Driver works!!!!!

### DISCUSSION THREAD

**Discussion by ghost** [copy/v86#525] [Show and tell]
so i tried an old Windows 98 hdd image that i made for Bochs, it takes long to boot and show desktop, but it plays startup sound and desktop appears!
download v86 state here:
https://www.mediafire.com/file/hj5465c9qgdjutn/win98se.bin/file

download full hdd image here: https://archive.org/download/bochs_windows_images/Windows%2098.zip

![image](https://user-images.githubusercontent.com/71384075/133410543-64d87eb6-55b6-46e0-9594-7ab7b29d819b.png)

**Comment by ghost** [comment]
Note: ne2k driver is not installed, you might need to copy windows 98 installation files using PowerISO and install Ne2k driver to get internet working, i also have Firefox installed on this disk image!

**Comment by ghost** [comment]
stuck here for 1 hour
![image](https://user-images.githubusercontent.com/71384075/133600911-e96f1fa0-1d46-49bc-8ead-663ad13f2479.png)

**Comment by MRVAPORWAVE25** [comment]
i this this one file v86 was glitching the  icons of my taskbar, desktop, and search

**Comment by algsniper607607-jpg** [comment]
hello ,i want to say the hdd file is working fine but there is no any driver installed how I can make an local folder on bochs on run time that I can browse files and then install drivers and other apps ,and i want it real time files reading and write not just readonly image ...thanks
